node/node.go

```diff
@@ -704,6 +704,14 @@ func (n *Node) WSAuthEndpoint() string {
 	return "ws://" + n.wsAuth.listenAddr() + n.wsAuth.wsConfig.prefix
 }
 
+// JWTPath returns the path for JWT secret
+func (n *Node) JWTPath() string {
+	if n.config.JWTSecret == "" {
+		return n.ResolvePath(datadirJWTKey)
```

Comment:
IMO, we should use a JWT token outside of the `nitro` directory, so that database copies and snapshots don't also copy the token.